src/transforms/coerce.jl

```diff
@@ -9,9 +9,15 @@ Return a copy of the table, ensuring that the scientific types of the columns ma
 
 This transform wraps the ScientificTypes.coerce function. Please see their docstring for more details.
 
+# Examples
+
 ```julia
 Coerce(:col1 => Continuous, :col2 => Count)
```

Comment:
```suggestion
T = Coerce(:col1 => Continuous, :col2 => Count)
```
Just for consistency.